src/Razor/test/Microsoft.NET.Sdk.Razor.Test/Microsoft.NET.Sdk.Razor.Test.csproj

```diff
@@ -13,19 +13,40 @@
     <!-- Copy references locally so that we can use them in the test. -->
     <CopyLocalLockFileAssemblies>true</CopyLocalLockFileAssemblies>
     <BuildVariablesGeneratedFile>$(MSBuildProjectDirectory)\obj\BuildVariables.generated.cs</BuildVariablesGeneratedFile>
+    <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
   </PropertyGroup>
 
   <ItemGroup>
     <None Include="xunit.runner.json" CopyToOutputDirectory="PreserveNewest" />
     <Compile Include="$(BuildVariablesGeneratedFile)" Condition="Exists('$(BuildVariablesGeneratedFile)')" />
+    <Compile Include="..\..\src\Microsoft.NET.Sdk.Razor\GenerateAspNetCoreStaticAssetsManifest.cs" Link="TasksSource\GenerateAspNetCoreStaticAssetsManifest.cs" />
```

Comment:
Didn't find any specific tests for the MSBuild tasks, I bundled them in here and just referenced the file with a link

Reply:
Seems good.